cristin-biobank/src/main/java/no/unit/nva/biobank/common/DomainUriUtils.java

```diff
@@ -0,0 +1,59 @@
+package no.unit.nva.biobank.common;
+
+import static nva.commons.core.paths.UriWrapper.HTTPS;
+import java.net.URI;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import nva.commons.core.StringUtils;
+import nva.commons.core.paths.UriWrapper;
+
+public final class DomainUriUtils {
+
+    private static final String BIOBANK_PATH_ELEMENT = "biobank";
+
+
+    private DomainUriUtils() {
+    }
+
+    public static URI getBiobankUri(String domainName, String basePath) {
+        return getBaseUriWrapper(domainName, basePath).addChild(BIOBANK_PATH_ELEMENT).getUri();
+    }
+
+    /**
+     * Building up URI ID for Biobank.
+     * @param domainName - env
+     * @param basePath - path
+     * @param identifier - code
+     */
+    public static URI getBiobankUri(String domainName, String basePath, String identifier) {
```

Comment:
Just wondering. Why do you need both `getBiobankUri()` and `getBiobankParamUri()`? Will the path to a biobank be different than `BIOBANK_PATH_ELEMENT` in any cases?